packages/orchestrator-ui-components/src/contexts/ApiClientContext.tsx

```diff
@@ -1,32 +1,25 @@
-import React, { createContext, useState, useEffect } from 'react';
+import React, { createContext, useContext } from 'react';
 import type { ReactNode } from 'react';
 import { ApiClient, getApiClient } from '../api';
+import { OrchestratorConfigContext } from './OrchestratorConfigContext';
 
 interface ApiContext {
     apiClient: ApiClient;
 }
 
 export const ApiClientContext = createContext<ApiContext>({
-    apiClient: getApiClient('https://orchestrator.dev.automation.surf.net/api'),
+    apiClient: getApiClient('http://localhost:8080/api'),
```

Comment:
This value can get set to ''. The default is already set in the config

Reply:
Done